package.json

```
@@ -18,12 +18,16 @@
   },
   "main": "dist/js/index.js",
   "scripts": {
-    "build": "yarn build:sdk && yarn downlevel && yarn build:tools",
+    "build": "yarn build:sdk && yarn downlevel && yarn build:tools && yarn build:plugin",
     "build:sdk": "tsc -p tsconfig.build.json",
     "build:sdk:watch": "tsc -p tsconfig.build.json -w --preserveWatchOutput",
     "build:tools": "tsc -p tsconfig.build.tools.json",
+    "build:plugin": "expo-module build plugin",
```

```suggestion
    "build:plugin": "EXPO_NONINTERACTIVE=true expo-module build plugin",
```

Let's keep it without watchdog by default like the other commands.